README.md

```diff
@@ -310,25 +162,31 @@ apt install \
   libjack-jackd2-dev # for JACK2, or libjack-dev for JACK1
 ```
 
-##### FreeBSD
-You may skip to "Compilation" - "Linux / BSD" - "FreeBSD" if you want to build via FreeBSD ports.
+##### Arch Linux/Manjaro/Artix:
+**TODO** 
 
-### Compilation
-Generally, BambooTracker releases are built like this:
+- check out AUR package and request fixing it if needed
```

Comment:
I believe the AUR packaging as been broken and outdated for awhile but I don't use Arch (btw) and didn't see the point in registering on the AUR just to comment on its brokenness.

Reply:
Perhaps, I can remove it for time being but still I would like to have mention on working AUR package for easier installation on btwOS-based distros

Reply:
@jpcima, iirc you are a maintainer of AUR package of BT, would you mind fixing and updating it?